config/proto/probe.proto

```diff
@@ -44,26 +44,15 @@ message HermesProbeDef {
   // Probes will not retry, they will fail if they timeout.
   optional int32 timeout_sec = 5;  // Timeout in seconds, default = 60
 
-  // If specified, latency is stored as a distribution
+  // If specified, latency is stored as a distribution metric.
```

Comment:
Note the measurement unit in the comments of distribution metric fields.

Reply:
Ah I do, but it's just slightly lower down 😛 in the code. 

"// The measurement unit of latency will be seconds."
on lines 49 and 53.